### ISSUE TITLE
[Grid] Keyboard navigation sample is missing

### ISSUE THREAD

**Issue by gedinakova** [IgniteUI/igniteui-blazor-examples#1088]
1. Navigate to Grid > Keyboard navigation [demo](https://staging.infragistics.com/products/ignite-ui-blazor/blazor/components/grids/grid/keyboard-navigation#custom-keyboard-navigation)

Result: The sample is not loaded

Expected result: The sample should either be loaded or the demo section should be hidden
